public/v2/apps/discourse.json

```diff
@@ -0,0 +1,127 @@
+{
+    "captainVersion": "2",
+    "documentation": "Taken from https://hub.docker.com/r/bitnami/discourse",
+    "displayName": "Discourse",
+    "description": "Discourse is the 100% open source discussion platform built for the next decade of the Internet. Use it as a mailing list, discussion forum, long-form chat room, and more!",
+    "dockerCompose": {
+        "version": "2",
+        "services": {
+            "$$cap_appname-postgresql": {
+                "documentation": "Taken from https://hub.docker.com/r/bitnami/postgresql",
+                "image": "docker.io/bitnami/postgresql:$$cap_postgresql_version",
+                "notExposeAsWebApp": "true",
+                "restart": "always",
+                "environment": { "ALLOW_EMPTY_PASSWORD": "yes" },
+                "volumes": ["$$cap_appname-postgresql-data:/bitnami/postgresql"]
+            },
+            "$$cap_appname-redis": {
+                "documentation": "Taken from https://hub.docker.com/r/bitnami/redis",
+                "image": "docker.io/bitnami/redis:$$cap_redis_version",
+                "notExposeAsWebApp": "true",
+                "restart": "always",
+                "environment": { "ALLOW_EMPTY_PASSWORD": "yes" },
+                "volumes": ["$$cap_appname-redis-data:/bitnami"]
+            },
+            "$$cap_appname": {
+                "documentation": "Taken from https://hub.docker.com/r/bitnami/discourse",
+                "image": "docker.io/bitnami/discourse:$$cap_discourse_version",
+                "containerHttpPort": "3000",
+                "depends_on": [
+                    "$$cap_appname-postgresql",
+                    "$$cap_appname-redis"
+                ],
+                "volumes": ["$$cap_appname-data:/bitnami"],
+                "environment": {
+                    "POSTGRESQL_HOST": "srv-captain--$$cap_appname-postgresql",
+                    "POSTGRESQL_ROOT_USER": "postgres",
+                    "POSTGRESQL_CLIENT_CREATE_DATABASE_NAME": "$$cap_db_name",
+                    "POSTGRESQL_CLIENT_CREATE_DATABASE_USERNAME": "$$cap_db_user",
+                    "POSTGRESQL_CLIENT_CREATE_DATABASE_PASSWORD": "$$cap_db_pass",
+                    "DISCOURSE_USERNAME": "$$cap_TODO",
+                    "DISCOURSE_PASSWORD": "$$cap_TODO",
+                    "DISCOURSE_EMAIL": "$$cap_TODO",
+                    "DISCOURSE_SITENAME": "$$cap_sitename",
+                    "DISCOURSE_POSTGRESQL_NAME": "$$cap_db_name",
+                    "DISCOURSE_POSTGRESQL_USERNAME": "$$cap_db_user",
+                    "DISCOURSE_POSTGRESQL_PASSWORD": "$$cap_db_pass",
+                    "DISCOURSE_HOST": "3000",
+                    "DISCOURSE_HOSTNAME": "$$cap_appname.$$cap_root_domain"
+                }
+            },
+            "$$cap_appname-sidekiq": {
+                "documentaion": "Taken from https://hub.docker.com/r/bitnami/discourse",
+                "image": "docker.io/bitnami/discourse:$$cap_discourse_version",
```

Comment:
Remove this line. `image` exists, hence `dockerfileLines` gets ignored by CapRover.